tests/libs/estdlib/test_gen_statem.erl

```diff
@@ -72,26 +88,45 @@ test_info() ->
 
 test_timeout() ->
     {ok, Pid} = gen_statem:start(?MODULE, [], []),
-
+    check_m_queue(?LINE),
     ok = gen_statem:call(Pid, {go_to_jail, 250}),
+    check_m_queue(?LINE),
     no = gen_statem:call(Pid, are_you_free),
+    check_m_queue(?LINE),
     timer:sleep(251),
+    check_m_queue(?LINE),
     yes = gen_statem:call(Pid, are_you_free),
+    check_m_queue(?LINE),
 
+    check_m_queue(?LINE),
     ok = gen_statem:call(Pid, {go_to_jail, 250}),
+    check_m_queue(?LINE),
     no = gen_statem:call(Pid, are_you_free),
+    check_m_queue(?LINE),
     ok = gen_statem:cast(Pid, escape),
     yes = gen_statem:call(Pid, are_you_free),
+    check_m_queue(?LINE),
     timer:sleep(251),
+    check_m_queue(?LINE),
     0 = gen_statem:call(Pid, get_num_spurious_timeouts),
+    check_m_queue(?LINE),
 
     gen_statem:stop(Pid),
+    check_m_queue(?LINE),
     ok.
 
+check_m_queue(Line) ->
+    receive
+        X -> io:format("check_m_queue, got ~p, line ~B\n", [X, Line])
```

Comment:
Will this result in output running the test?

Reply:
Oops